handlers.go

```diff
@@ -318,21 +312,29 @@ func (dht *IpfsDHT) handleGetProviders(ctx context.Context, p peer.ID, pmes *pb.
 	providers := dht.providers.GetProviders(ctx, c)
 	if has {
 		providers = append(providers, dht.self)
-		log.Debugf("%s have the value. added self as provider", reqDesc)
+		log.LogKV(ctx,
+			"event", "adding self as a provider",
+		)
```

Comment:
I am not sure there is any readability benefit from having this multiline...

Reply:
So, I made it multiline to make the key/value relationship clear. I've now converted the rest of the module to the new code so I'd like you to take another look, when you get time.